source/guide_timetagger.rst

```diff
@@ -0,0 +1,175 @@
+  […]
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Create main file
+----------------
+
+To configure TimeTagger we need to create our own main file.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ cp ~/.local/lib/python3.10/site-packages/timetagger/__main__.py ~/bin/mytimetagger.py
+ [isabell@stardust ~]$
+
+Next we select the port ``4800`` to serve TimeTagger from. It must be mapped in the backend.
+
+.. include:: includes/web-backend.rst
+
+First Run
+---------
+
+We can now start the app for the first time. Therefor replace the number after the colon with the port number you chose.
+
+Right now a user account is still missing. The app provides a dialogue to generate the encrypted string where a user is stored in.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ python3.10 ~/bin/mytimetagger.py --bind=0.0.0.0:4800
+ […]
+ [INFO 2022-07-08 12:14:34] Collected 4 assets from /home/isabell/.local/lib/python3.10/site-packages/timetagger/pages/.
+ [INFO 2022-07-08 12:14:34] Server is starting up
+
+Finishing installation
+======================
+
+Now we can browse to https://isabell.uber.space/timetagger/cred to generate our preferred credentials. Choose a strong password. As a result you receive a string like ``isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa``.
+We use this string and test the login by adding the credential to the startup script.
+
+First stop the running python script by sending ``CTRL+C`` to the console.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [INFO 2022-07-08 12:14:34] Server is starting up
+ ^C[INFO 2022-07-08 12:44:39] Server is shutting down
+ [isabell@stardust ~]$
+
+
+.. note:: The credentials must be encapsulated by single quotation marks.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ python3.10 ~/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
+ […]
+ [INFO 2022-07-08 12:46:34] Collected 4 assets from /home/isabell/.local/lib/python3.10/site-packages/timetagger/pages/.
+ [INFO 2022-07-08 12:46:34] Server is starting up
+
+We should now be able to successfully login. Finally we setup a deamon to run TimeTagger automatically.
+
+Set up the deamon
+-----------------
+Create the file ``~/etc/services.d/timetagger.ini`` with the following content:
+
+.. note:: Replace the port and credentials with the credentials you previously created. Also, make sure to enter the correct home directory instead of isabell.
+
+.. code-block:: ini
+ :emphasize-lines: 2
+
+ [program:timetagger]
+ command=python3.10 /home/isabell/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
```

Comment:
We can just use the home variable so we have fewer isabells here.
```suggestion
.. note:: Replace the port and credentials with the credentials you previously created.

.. code-block:: ini
 :emphasize-lines: 2

 [program:timetagger]
 command=python3.10 %(ENV_HOME)s/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
```

Reply:
As the user must replace the credentials anyways I'd prefer to keep it simple for the average joe and have him just replace `isabell` twice.

Reply:
Please use variables whenever possible